pkg/function/create_volume_from_snapshot.go

```diff
@@ -125,6 +128,22 @@ func createVolumeFromSnapshot(ctx context.Context, cli kubernetes.Interface, nam
 	return providerList, nil
 }
 
+func addPVProvisionedByAnnotation(annotations map[string]string, provider blockstorage.Provider) map[string]string {
```

Comment:
Do we really need `annotations` arg in the function? The only reference I found passes `nil` value to this argument. Can we get rid of it?

Reply:
I added annotations just for the cases where we have a PV's annotations and want to set this annotation. Not being used right now, but might be used in future.